content/en/docs/reference/command-line-tools-reference/feature-gates/recover-volume-expansion-failure.md

```diff
@@ -9,6 +9,9 @@ stages:
   - stage: alpha 
     defaultValue: false
     fromVersion: "1.23"
+  - stage: beta
```

Comment:
`toVersion` should be added to the previous stage.

```suggestion
    toVersion: "1.31"
  - stage: beta
```

Reply:
Thanks for the remark